src/codegen/compile.ml

```diff
@@ -3671,6 +3672,14 @@ module IC = struct
         edesc = nr (FuncExport (nr (E.built_in env "print_ptr")))
       })
 
+
+  let performance_counter env =
+    match E.mode env with
+    | Flags.ICMode | Flags.RefMode ->
```

Comment:
```suggestion
    | Flags.(ICMode | RefMode) ->
```

Reply:
We use this style elsewhere...

Reply:
My conception is that it makes the two relevant pieces  of information appear nearer and is thus better digestible. But I am not radical enough about it to change all occurrences.